test/targets/python/requests.js

```diff
@@ -1,6 +1,16 @@
+/* global it */
+
 'use strict'
 
 require('should')
 
-module.exports = function (snippet, fixtures) {
+module.exports = function (HTTPSnippet) {
+  it('should support query parameters provided in HAR\'s url', function () {
+    var result = new HTTPSnippet({ 'method': 'GET', 'url': 'http://mockbin.com/har?param=value' }).convert('python', 'requests', {
+      showBoilerplate: false
+    })
+
+    result.should.be.a.String()
+    result.should.eql('import requests\n\nurl = "http://mockbin.com/har"\n\nquerystring = {"param":"value"}\n\nresponse = requests.request("GET", url, params=querystring)\n\nprint(response.text)')
```

Comment:
looks good. only feedback is to make output human readable using backticks. I refactored the `go` tests recently you can use as an example. This helps with git diffs in a future PR if tests fail.

https://github.com/Kong/httpsnippet/blob/c8e0b09fa3b9db4a75c29de85470851b04745801/test/targets/go/native.js#L22

Reply:
Done :)